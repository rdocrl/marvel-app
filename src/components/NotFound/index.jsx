import notFound from '../../assets/not-found.png';

const NotFound = () => {
  return (
    <>
      <h2>404 - Not found</h2>
      <img src={notFound} alt="oop - 404 not found" width={300} />
    </>
  );
};

export default NotFound;
