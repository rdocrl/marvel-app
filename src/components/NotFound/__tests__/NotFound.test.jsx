import { render } from '@testing-library/react';
import NotFound from '../';

describe('<NotFound /> tests', () => {
  describe('Snapshots', () => {
    it('Should render', () => {
      const { asFragment } = render(<NotFound />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
