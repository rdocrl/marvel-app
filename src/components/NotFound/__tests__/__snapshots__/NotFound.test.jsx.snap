// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> tests Snapshots Should render 1`] = `
<DocumentFragment>
  <h2>
    404 - Not found
  </h2>
  <img
    alt="oop - 404 not found"
    src="not-found.png"
    width="300"
  />
</DocumentFragment>
`;
